packages/web-components/src/components/card/__stories__/card.stories.ts

```diff
@@ -45,6 +46,53 @@ const iconPlacement = {
   right: 'right',
 };
 
+const pictogramPlacements = {
+  [PICTOGRAM_POSITION.TOP]: PICTOGRAM_POSITION.TOP,
+  [PICTOGRAM_POSITION.BOTTOM]: PICTOGRAM_POSITION.BOTTOM,
+};
+
+export const Pictogram = ({ parameters }) => {
+  const { href, heading, copy, pictogramPlacement, eyebrow, inverse } = parameters?.props?.PictogramCard ?? {};
+  return html`
+    <dds-card
+      color-scheme=${inverse ? 'inverse' : ''}
+      pictogram-position="${pictogramPlacement}"
```

Comment:
To align to `ICON_PLACEMENT` for `<dds-card-footer>`:

```suggestion
      pictogram-placement="${pictogramPlacement}"
```